application-templates/starter-typescript/src/components/channel-details/channel-details.spec.tsx

```diff
@@ -8,16 +8,13 @@ import {
   mapResourceAccessToAppliedPermissions,
   type TRenderAppWithReduxOptions,
 } from '@commercetools-frontend/application-shell/test-utils';
-import { reportErrorToSentry } from '@commercetools-frontend/sentry';
```

Comment:
We use sentry internally (not sure if a customer uses that), no need to have it included by default in the template then.